packages/react-router-dom/__tests__/data-static-router-test.tsx

```diff
@@ -857,7 +857,7 @@ describe("A <StaticRouterProvider>", () => {
           {
             path: "path",
             element: <h2>Hi again!</h2>,
-            errorElement: <h2>Error again!</h2>,
+            ErrorBoundary: () => <h2>Error again!</h2>,
```

Comment:
Do you mind leaving this test as-is and copy/pasting a new version using `ErrorBoundary`?  That way both are properly tested.

Reply:
So with current changes such combination are present in tests (parent-child):
`errorElement-ErrorBoundary`
`ErrorBoundary-errorElement`
`lazy(errorElement)-lazy(ErrorBoundary)`
`errorElement-none`
`lazy(errorElement)-none`

You want me to copy this test keeping it as `errorElement-errorElement` and paste it with `ErrorBoundary-ErrorBoundary`?

Reply:
yeah - we can just duplicate the existing `handles framework agnostic static handler routes` test and change the new one to use `ErrorBoundary`:

```
// current test
it("handles framework agnostic static handler routes", () => {...})

// copy/pasted new test
it("handles framework agnostic static handler routes (using ErrorBoundary)", () => {...})
```

Reply:
Done, also fixed preexisting typo in that test, instead of `hasErrorBoundary` in agnostic routes it was `hasErrorElement` (wasn't affecting anything).